regolith/helpers/u_milestonehelper.py

```diff
@@ -13,28 +13,29 @@
 ALLOWED_STATI = {"p":"proposed", "s":"started", "f":"finished", "b":"back_burner","c":"converged"}
 
 def subparser(subpi):
-    subpi.add_argument("projectum_id", help="the id of the projectum")
+    subpi.add_argument("projectum_id", help="The id of the projectum.")
     subpi.add_argument("-v", "--verbose", action="store_true",
-                        help="gives a list of the milestones available to update.")
+                        help="Gives a list of the milestones available to update "
+                             "with specific information about each milestone.")
     subpi.add_argument("-i", "--index",
-                        help="index of the item in the enumerated list to update",
+                        help="Index of the item in the enumerated list to update. ",
                         type = int)
     subpi.add_argument("-d", "--due_date",
-                       help="new due date of the milestone in ISO format (YYYY-MM-DD) "
-                            "required to add a new milestone")
+                       help="New due date of the milestone in ISO format(YYYY-MM-DD). "
+                            "Required to add a new milestone. ")
     subpi.add_argument("-n", "--name",
-                       help="name of the new milestone. "
-                            "required to add a new milestone.")
+                       help="Name of the new milestone. "
```

Comment:
"Name of the milestone. Required for a new milestone."

Reply:
done!